### ISSUE TITLE
詳解AI公司三大類別，哪種商業模式將成為最後贏家？

### ISSUE THREAD

**Issue by sbinlin** [twaisu/General#7]
[original link](http://bangqu.com/mP8QTE.html)
The 3 major categories of AI companies

The jury is still out on who the biggest AI winners in the enterprise space will be. So far, applying AI to enterprise has not made as much impact as people have expected. Cloud computing, for instance, has had far greater impact in the enterprise space than AI has. The CEO of Databricks puts this issue well:

AI has enormous promise but also a 1% problem. Less than 10 companies in the world are achieving the full potential of AI and the rest are really struggling.
There’s a huge opportunity to help other enterprises unlock the full potential of AI. Databricks, a company whose roots came from helping enterprises with big data processing, recently took a $140M round to make “Artificial Intelligence (AI) achievable for enterprise organizations with its Unified Analytics Platform.” Databricks has a unique advantage in their recent move to become an AI platform company. That said, I believe that the vast majority of successful AI companies in the future will be vertical AI companies.

The vast majority of successful AI companies in the future will be vertical AI companies as opposed to AI platform companies.
First, let’s define a few terms. AI companies can be placed into three major categories: data science consulting firms, AI platform companies, and vertical AI companies. These categories can be defined by two different measures. The first measure is the level of productization (low or high). The second measure is the number of use cases that the solution targets (few or many). Each of the categories can be defined by these measures.

Data science consulting firms: low productization
AI platform companies: high productization targeting many use cases
Vertical AI companies: high productization targeting few use cases

Being in each of these categories has their unique advantages and drawbacks. There will be winners in each category, but I expect the majority of winners will be vertical AI companies. Let’s explore each of these categories further.

Data Science Consulting Firms
Data science consulting firms are defined by their low level of productization. Their main advantage is that it’s easier for them to deliver great results, as AI models require customization and are highly dependent on customer data. Their disadvantage is that they cannot scale quickly. For companies that are expected to be high growth, they will need to figure out how to move out of this category.

There are more companies in this category than may first appear. Many AI companies do not realize or worry that they fall into this category — until they have trouble scaling or finding repeatable use cases. They may have unique technology, gain customers, and even receive significant acquisition offers. But to scale beyond a certain size, they must “grow up” and productize their solutions [1]. Unfortunately, figuring out how to “grow up” is not easy in many cases. Productizing AI solutions can be extremely difficult for two main reasons. As mentioned before, many AI solutions are extremely dependent on the data, which can vary widely across customers. Further, the precise need across customers may differ just enough to require significant customization. (More in a future post.)

An example of an AI company that can be defined as a “data science consulting firm” is Element AI, which raised a giant Series A round of $102M earlier this summer. As of now, they do not have a specific product or solution offering. Their Solutions Portfolio page lists 15 different use cases, and their Solutions page describes that they “build customized applications that are easy to integrate into existing processes.” Due to their huge investment round, Element AI will be expected to “grow up” and productize their AI solutions in the future.

What exactly does productization mean? Solutions are productized if they can be leveraged across multiple installments with little customization, enabling the company to scale revenue at a rate that greatly outpaces incremental cost increases. In contrast, the number of employees a consulting firm must hire is roughly linear to the number of concurrent engagements that the firm has.

Luckily for Element AI and others, full productization (i.e. no customization per installment) is not required. For example, a particular AI company may decide to invest engineering resources into features that can decrease the manual cost of each installment by 80%. Reaching this 80% decrease may require just 20% of the engineering effort compared to reaching full productization. The cost of the remaining 20% effort could be passed on to the client as a reasonable one-time installation fee. Targeting this 80% productization goal therefore may be enough to achieve enough product leverage to scale.

There are other benefits to productization. Productization can create barriers to entry for newcomers who would similarly struggle to scale, lead to much faster integration times, and allow the technical team to focus on improving the core IP of the company.

AI Platform companies
AI platform companies offer to be the underlying infrastructure on top of which specific AI solutions live. They can allow end users to import data, perform data wrangling and transformations, train models, and perform model validation. Examples include H2O.ai, Skytree, and (more recently) Databricks.

These companies have a number of unique advantages. For one, their total addressable market is huge. They can potentially sell to any enterprise that is looking to add to their AI arsenal. In addition, AI platform companies tend to be highly productized as they are only dependent on the underlying computation framework, OS, and hardware that supports them. Moreover, they have a stickiness factor, meaning that once an enterprise relies on the platform, it would be difficult to get rid of.

However, the viability of most AI platform companies is threatened by increases in:

the number and quality of open source AI projects
the number of programmers who know how to use them
The growing optionality and quality of open source projects in recent years is astounding. Google’s TensorFlow library, which started off for deep learning only, has since expanded and gained wide-scale acceptance for training, testing, and maintaining machine learning models. With TensorFlow, it’s easy to quickly train a model with good performance. But it also has many features to deploy models in a production environment (e.g. TensorFlow Serving). Many other companies and individuals contribute to the open source environment as well. Yahoo has an open source project to run TensorFlow on top of Apache Spark. Facebook open-sourced their internal AI platform, FBLearner Flow, and made many contributions to the open-source Torch machine learning library. Scikit-learn is another popular AI library, first created in 2007 by developer David Cournapeau. It now has over 900 contributors to the project.

The number of programmers who are comfortable using these open source tools is also increasing. Supply is fueled by demand, and in the next five years, demand for data science jobs is expected to increase by 15%. These data scientists will be familiar with the standard open source AI tools, which are taught in data science courses. This is different from the old guard of analysts who learned to manipulate data and build models in SAS, Excel, Alteryx, MATLAB, and other closed platforms.

With these two major forces, most enterprises would choose open source tools and optionally a consultant to help implement them, leaving true AI platform companies with significantly less to collect on.

There are already a few signs that AI platform companies are not doing well. H2O.ai laid off 10% of their workforce in Sept 2016 to switch from selling to many clients to focusing on a select few. They are likely going down the productization axis, delivering solutions that are more bespoke but will yield bigger contracts (with a smaller sales team). They may also decide to focus on a particular set of use cases and eventually become a vertical AI company, not an uncommon path.

Databricks may be able to create a compelling AI platform offering due to their world-class expertise of Apache Spark. For instance, they may be able to make significant improvements to Spark specifically for AI that would be difficult for others to replicate. Without significant differentiation from open source solutions, Databricks would have to compete with in-house teams and other companies who implement open source solutions (e.g. IBM’s Data Science Experience, which delivers Jupyter notebooks on top of Spark). As another alternative, Databricks may focus on delivering vertical AI solutions as their main source of revenue.

Vertical AI Companies
Vertical AI companies solve a particular business problem or set of problems with a productized solution. They enable their enterprise customers to achieve additional lift from AI without needing to build or maintain models in-house. Examples on this end are more numerous. Some include DigitalGenius for customer support, Entelo for recruiting, Cylance for cybersecurity, and DataVisor (where I work) for fraud detection.

Earlier this year, Andrew Ng made the case for why AI is the new electricity [2]. In the same article, he writes:

Just as electricity transformed almost everything 100 years ago, today I actually have a hard time thinking of an industry that I don’t think AI will transform in the next several years.
Not only will AI power many of the core technologies offered by existing enterprises, it will also power supporting operations. These include sales, marketing, customer support, internal communication, finance, and more. Because these supporting groups are not core IP, enterprises will not prioritize developing proprietary AI solutions to improve them. Instead, they’ll buy such solutions. This is the opportunity for vertical AI companies.

Thus, vertical AI companies can leverage a productized solution to scale and do not face the same existential questions that an AI platform company does. Disadvantages of vertical AI companies are that they may face more competition from other companies and their total addressable market is smaller than a platform AI company. But if Andrew Ng and others are right, the growing pie would be big enough for many vertical AI company winners in all markets. This is why I believe the vast majority of winners in the AI enterprise space will be vertical AI companies.

Of course, not all vertical AI companies will be winners. There are 3 major things that a vertical AI company needs.

First, the solution must have a strong business case that solves a real problem. There exist vertical AI companies that use buzz words to catch attention, when in reality their solution does not obtain significant benefit from AI. For companies that do have a true AI solution, their challenge is articulating the differentiated benefits of their solution to a less technically sophisticated audience.

Second, the solution must not compete with an in-house team, now nor in the future. Trying to sell an AI solution when there is already an in-house team to address the problem naturally makes the sale much more difficult to near impossible. Further, trying to sell an AI solution to an enterprise that may want to implement their own solution in the future is a long-term risk. The enterprise may learn from the vertical AI company and then kick them out, replacing the solution with their in-house team.

Third, the integration effort should be reasonable. The definition of “reasonable” depends on the ROI from the solution and also who the end customer is. For some clients, a “reasonable” integration time is one day while for others, it’s one year if the solution has high ROI. Integration effort is less of an issue if the solution does not have any major dependencies. But for solutions that depend on client data or infrastructure, this is important to consider.

Looking forward
Even today, AI is more pervasive than many think. Netflix recommendations, Amazon’s Alexa, and credit card fraud alerts are just a few examples that touch millions of consumers. Looking forward to the next ten years, benefits derived from AI solutions will accelerate as people figure out the best ways to create and become AI companies.

And after that, the term “AI company” will be redundant.

Notes
There are other reasons why many AI companies never “grow up” besides the difficulty in productizing AI solutions. Shivon Zilis and James Cham from Bloomberg Data make the case that it’s due to an absence of a market. Such a problem would be lessen as AI becomes a more familiar topic. They also make the case, along with Victor Basta from Magister Advisor, that few companies realize the potential of many up-and-coming AI companies —but when it happens, they’d rather acquire the team and technology rather than allow competitors benefit from the AI solution. The reality is likely to be a blend of these and other factors, depending on the precise industry, use case, and profile of the end consumer.
I owe a lot to Professor Ng. The first AI class I took was his Machine Learning course (CS229). Taking that class changed my academic and career trajectories in a profound manner.
